### PR TITLE
Add default keymaps for Clawsome keyboards

### PR DESCRIPTION
**Pull request by AlisGraveNil**
Keymaps for clawsome/bookerboard and clawsome/sidekick

### CODE REVIEW

File: public/keymaps/c/clawsome_bookerboard_default copy.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (noroadsleft) [inline public/keymaps/c/clawsome_bookerboard_default copy.json:1]: Just realized this file doesn't have the correct filename.

```shell
git mv -- "public/keymaps/c/clawsome_bookerboard_default copy.json" "public/keymaps/c/clawsome_bookerboard_default.json"
git commit -m "renamed 'clawsome_bookerboard_default copy.json' to 'clawsome_bookerboard_default.json'"
```

Comment (AlisGraveNil) [reply public/keymaps/c/clawsome_bookerboard_default copy.json:1]: Are those commands I need to do, or did you just do them?

Comment (noroadsleft) [reply public/keymaps/c/clawsome_bookerboard_default copy.json:1]: You need to do these in Git through your command line interface.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**